packages/flight_controller_driver/src/flight_controller_node.py

```diff
@@ -251,25 +266,31 @@ def _srv_calibrate_imu_cb(self, _):
         self.loginfo("Calibrating IMU...")
         try:
             with self._lock:
-                self._board.sendCMD(0, MultiWii.ACC_CALIBRATION, [])
-                self._board.receiveDataPacket()
-                # wait 2 secs
-                time.sleep(2)
+                if self._board.send_RAW_msg(MSPy.MSPCodes['MSP_ACC_CALIBRATION'],data=[]):
+                    dataHandler = self._board.receive_msg()
+                    self._board.process_recv_data(dataHandler)
                 # read IMU 10 times
                 data = copy.deepcopy(self._default_accelerometer_calib)
                 counter = 1
                 num_points = 20
+                
                 for _ in range(num_points):
                     # noinspection PyBroadException
                     try:
-                        self._board.getData(MultiWii.RAW_IMU)
-                        data["x"] += self._board.rawIMU["ax"]
-                        data["y"] += self._board.rawIMU["ay"]
-                        data["z"] += self._board.rawIMU["az"]
-                        counter += 1
-                        time.sleep(1.0 / self._frequency["imu"])
-                    except Exception:
-                        pass
+                        self._board.fast_read_imu()
+                        time.sleep(0.2)
```

Comment:
can we do this more often? `20 * 200ms = 4s`

Reply:
It seems to drop more packages if done more often.

Reply:
define "drop packages"